src/app/views/canvas/mark_editor.tsx

```diff
@@ -706,7 +707,7 @@ export class SingleMarkView
   public renderHandles() {
     return (
       <g>
-        {this.renderMarkHandles()}
+        {/* this.renderMarkHandles() */}
```

Comment:
Do we need this anymore?

Reply:
I was treating it the same as `this.renderAnchorHandles()` below